.pre-commit-config.yaml

```diff
@@ -2,20 +2,20 @@ ci:
   autofix_prs: false
 repos:
   - repo: https://github.com/pre-commit/pre-commit-hooks
-    rev: v4.6.0
+    rev: v5.0.0
```

Comment:
Does the newest version of pre-commit-hooks/astral/black get pulled in  and updated in this file and automatically, or do we just check for updated versions periodically?

Reply:
so there was an automated pr #32 that was failing snyk. And this PR fixed that so I just cut out the middleman and ran `pre-commit autoupdate` which did this and then I checked it in.